gen/solid.hoon

```diff
@@ -60,7 +60,9 @@
 =+  txt=.^(@ %cx (weld pax `path`[%hoon ~]))
 =+  sam=[now `ovum`[[%gold ~] [%veer p.i.vay pax txt]]]
 ~&  [%solid-veer i.vay]
-=+  gat=.*(all .*(all [0 42]))
+::  +poke at +10
```

Comment:
I think all these invocations should be Nock 9s. Feel free to tell me this is out of scope, but it seems like it might be a good time to fix it. If we ever want to jet parts of the Arvo kernel, we'll need to use Nock 9s instead of this form.
```
=+  gat=.*(all [9 10 0 1])
```

Reply:
Hmm, maybe I was wrong about jetting. This still feels like a Nock 9 to me, though.

Reply:
I agree that these should be 9s. I have more updates to `+solid` planned, as I intend to get release-candidate arvo dual-booting on release-candidate and cc-release vere.